deepchem/feat/molecule_featurizers/molgan_featurizer.py

```diff
@@ -0,0 +1,248 @@
+import logging
+import numpy as np
+from deepchem.utils.typing import RDKitBond, RDKitMol, List
+from deepchem.feat.base_classes import MolecularFeaturizer
+
+logger = logging.getLogger(__name__)
+
+
+class GraphMatrix:
+  """
+  This is class used to store data for MolGAN neural networks.
+
+  Parameters
+  ----------
+  node_features: np.ndarray
+    Node feature matrix with shape [num_nodes, num_node_features]
+  edge_features: np.ndarray,
+    Edge feature matrix with shape [num_nodes, num_nodes]
+
+  Returns
+  -------
+  graph: GraphMatrix
+    A molecule graph with some features.
+  """
+
+  def __init__(self, adjacency_matrix: np.ndarray, node_features: np.ndarray):
+    self.adjacency_matrix = adjacency_matrix
+    self.node_features = node_features
+
+
+class MolGanFeaturizer(MolecularFeaturizer):
+  """Featurizer for MolGAN de-novo molecular generation [1]_.
```

Comment:
Need a space after the first line I believe

Reply:
which line, you mean after class?